app/tests/components_tests/test_serializers.py

```diff
@@ -246,6 +253,35 @@
 }
 
 
+def test_civ_post_validate_provided_fields():
+    values = [
+        {},  # no values is also valid
+        {"file": "https://some-api-url/"},
+        {"image": "https://some-api-url/"},
+        {"upload_session": "https://some-api-url/"},
+        {"user_upload": "https://some-api-url/"},
+        {"image_name": "foobar", "user_uploads": ["https://some-api-url/"]},
+        {"value": 42},
+    ]
+    payload_empty = {
+        "interface": "my-socket",
+        "file": None,
+        "image": None,
+        "image_name": None,
+        "upload_session": None,
+        "user_upload": None,
+        "user_uploads": None,
+        "value": None,
+    }
+
+    for value in values:
+        payload = {**payload_empty, **value}
+        with nullcontext():
+            ComponentInterfaceValuePostSerializer._validate_provided_fields(
+                attrs=payload
+            )
```

Comment:
```suggestion
        ComponentInterfaceValuePostSerializer._validate_provided_fields(
            attrs=payload
        )
```

Alternatively, it might be good to add a 'negative' test here, where you check that any invalidity is really captured. Then again, I think a later test does this indirectly already.